src/Umbraco.Web.UI.Client/src/packages/search/examine-management-dashboard/views/section-view-examine-searchers.ts

```diff
@@ -212,7 +239,19 @@ export class UmbDashboardExamineSearcherElement extends UmbLitElement {
 					</uui-tag>
 				</uui-icon-registry-essential>
 			</button>
-		</div>`;
+		</div>
+		<div style="resize: horizontal; overflow: hidden; padding: 6px;">
```

Comment:
Could we move this into the `styles` section rather than inlining it?  Maybe check all is necessary too.